components/UI/actions/clickable/clickableSocialIcon.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useEffect, useState } from 'react';
+import { Tooltip } from '@mui/material';
+import TwitterIcon from '@components/UI/iconsComponents/icons/twitterIcon';
+import GithubIcon from '@components/UI/iconsComponents/icons/githubIcon';
+import DiscordIcon from '@components/UI/iconsComponents/icons/discordIcon';
+import VerifiedIcon from '@components/UI/iconsComponents/icons/verifiedIcon';
+import styles from '@styles/components/icons.module.css';
+import theme from '@styles/theme';
+
+type SocialPlatform = 'twitter' | 'github' | 'discord';
+
+interface ClickableSocialIconProps {
+  platform: SocialPlatform;
+  width: string;
+  profileId: string;
+  domain?: string;
+}
+
+const ClickableSocialIcon: React.FC<ClickableSocialIconProps> = ({
+  platform,
+  width,
+  profileId,
+  domain,
+}) => {
+  const [githubUsername, setGithubUsername] = useState<string | undefined>();
```

Comment:
use a linter ! Here you're setting the username for nothing because you don't use `githubUsername` in the code. 

This means you can delete all the code relative to the github api too

Reply:
ohh my bad! 😅 Sorry about that! @fricoben Ive cleaned it up now. I'll keep an eye on those linters going forward. Thanks again!